### PR TITLE
Examples for Apache Kafka in Go

### PR DESCRIPTION
**Pull request by anelook**
# What changed, and why it matters

### CODE REVIEW

File: docs/products/kafka/howto/connect-with-go.rst [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+``HOST``                     Host name for the connection
+``USER_NAME``                Name of the user for the connection
+``SSL_PORT``                 Port number to use for SSL
+``SASL_PORT``                Port number to use for SASL
+``SASL_PASSWORD``            Password required to connect using SASL
+``CLIENT_ID``                Application-specific client id
+========================     =======================================================================================================
+
+With library ``sarama``
+-----------------------_
```
Comment (tvainika) [inline docs/products/kafka/howto/connect-with-go.rst:56]: This part is rendered oddly in review page.
image

File: docs/products/kafka/howto/connect-with-go.rst [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+---------
+
+========================     =======================================================================================================
+Variable                     Description
+========================     =======================================================================================================
+``HOST``                     Host name for the connection
+``USER_NAME``                Name of the user for the connection
+``SSL_PORT``                 Port number to use for SSL
+``SASL_PORT``                Port number to use for SASL
+``SASL_PASSWORD``            Password required to connect using SASL
```
Comment (tvainika) [inline docs/products/kafka/howto/connect-with-go.rst:51]: As `USER_NAME` and `SASL_PASSWORD` are always used together, maybe change the ordering such that those two are next to each other? Also variable name could be `SASL_USERNAME` to clarity.

Comment (anelook) [reply docs/products/kafka/howto/connect-with-go.rst]: Good idea! changed

File: docs/products/kafka/howto/connect-with-go.rst (hunk 1 above)
Comment (anelook) [reply docs/products/kafka/howto/connect-with-go.rst]: Thanks! didn't notice this. Fixed now

File: docs/products/kafka/howto/connect-with-go.rst [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+        tlsConfig := &tls.Config{
+            RootCAs: caCertPool,
+        }
+
+        // parse Kafka cluster version
+        version, err := sarama.ParseKafkaVersion("2.4.0")
+        if err != nil {
+            panic(err)
+        }
```
Comment (tvainika) [inline docs/products/kafka/howto/connect-with-go.rst:143]: This part looks odd. Not really needed at all? (also see later assignment of `config.Version = version` which is not there non-sasl snippet.

**Review by tvainika — APPROVED** [review]

**Pull request merged**